From 42d1076fb25432db3e408de845b80970a72649ab Mon Sep 17 00:00:00 2001
From: Michael Goffioul <michael.goffioul@lincor.com>
Date: Fri, 4 Oct 2019 20:56:44 -0400
Subject: [PATCH 4/7] Generate system root image, no ramdisk

The generated system partition is expected to be mounted at / and
includes the former ramdisk (from $OUT/root directory).
---
 BoardConfig.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index bd23daf..fac4810 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -89,3 +89,5 @@ DEVICE_MANIFEST_FILE := device/generic/common/manifest.xml
 
 BOARD_SEPOLICY_DIRS += device/generic/common/sepolicy \
                        system/bt/vendor_libs/linux/sepolicy \
+
+BOARD_BUILD_SYSTEM_ROOT_IMAGE:= true
-- 
2.21.0

